Misc/NEWS.d/next/Library/2018-08-04-12-26-11.bpo-28009.4JcHZb.rst

```diff
@@ -0,0 +1 @@
+uuid: correct the field seperator (_mac_field) value of an ethernet adapter MAC address for AIX
```

Comment:
This NEWS entry is misleading: your change includes other changes. Maybe you could just say "Fix uuid.uuid1() and uuid.get_node() on AIX"?

Reply:
done